query_server/query/src/sql/parser.rs

```diff
@@ -655,16 +686,23 @@ impl<'a> ExtParser<'a> {
         let database_name = self.parser.parse_identifier()?;
         self.parser.expect_keyword(Keyword::SET)?;
         let mut options = DatabaseOptions::default();
-        if !self.parse_database_option(&mut options)? {
+        let mut config = DatabaseConfig::default();
+        if !self.parse_database_option_or_config(&mut options, &mut config)? {
             return parser_err!(format!(
                 "expected database option, but found {}",
                 self.parser.peek_token()
             ));
         }
-        Ok(ExtStatement::AlterDatabase(AlterDatabase {
-            name: database_name,
-            options,
-        }))
+        if config.has_some() {
+            return parser_err!("database config is unmodifiable, only can modify database option：TTL, SHARD, VNODE_DURATION, REPLICA".to_string());
```

Comment:
Here has a full-width colon: `database option：TTL,`, while other colons are half-width.

Here has all of **database option**: `TTL, SHARD...`, while none of **database config**. Why not tell user what the database options are?

```
database config (PRECISION, MAX_MEMCACHE_SIZE, MEMCACHE_PARTITIONS, WAL_MAX_FILE_SIZE, WAL_FLUSH_TRIGGER_TOTAL_FILE_SIZE, WAL_SYNC, WAL_SYNC_INTERVAL, WAL_REQ_CHANNEL_CAP, STRICT_WRITE, MAX_CACHE_READERS) is unmodifiable, only database option(TTL, SHARD, VNODE_DURATION, REPLICA) is modifiable
```

Reply:
there are too many config, and config can't be modified, only tell user which can modify